manifests/kustomize/sample/kustomization.yaml

```diff
@@ -4,6 +4,11 @@ kind: Kustomization
 bases:
   # Or github.com/kubeflow/pipelines/manifests/kustomize/env/gcp?ref=1.0.0
   - ../env/gcp
+  # Kubeflow Pipelines servers are capable of collecting Prometheus metrics.
+  # If you want to monitor your Kubeflow Pipelines servers with those metrics, you'll need a Prometheus server in your Kubeflow Pipelines cluster.
+  # If you don't already have a Prometheus server up, you can use the following configuration files for Prometheus.
+  # If you have your own Prometheus server up already or you don't want a Prometheus server for monitoring, you can comment the following line out.
+  - third-party/prometheus
```

Comment:
Hmmm, I meant `../third-party/prometheus`, because this sample folder should only contains minimal last mile overrides.

Reply:
done